docs/pages/modules/existing-library.mdx

```diff
@@ -0,0 +1,111 @@
+---
+title: Intgerating in an existing library
+---
+
+import { CodeBlocksTable } from '~/components/plugins/CodeBlocksTable';
+import { Terminal } from '~/ui/components/Snippet';
+
+There are cases where you may want to integrate the Expo Modules API into an existing React Native library. For example, this may be useful to incrementally rewrite the library or to take advantage of [iOS AppDelegate subscribers](/modules/appdelegate-subscribers/) and [Android Lifecycle listeners](/modules/android-lifecycle-listeners/) to automatically set up your library.
+
+The following steps will set up your existing React Native library to access Expo Modules APIs.
+
+Create the [**expo-module.config.json**](./module-config.mdx) file at the root of your project and start from the empty object `{}` inside it. We will fill it in later to enable specific features.
+The presence of the module config is enough for [Expo Autolinking](/workflow/glossary-of-terms/#expo-autolinking) to recognize it as an Expo module and automatically link your native code.
```

Comment:
Since we now have a dedicated page for autolinking, let's use it instead.
```suggestion
The presence of the module config is enough for [Expo Autolinking](./autolinking.mdx) to recognize it as an Expo module and automatically link your native code.
```